src/concepts/glean/glean.md

```diff
@@ -1,46 +1,69 @@
-To make engineers more productive, the SDK keeps the amount of instrumentation code required for metrics as small as possible. Engineers only need to specify what they want to instrument, with which semantics and then record the data using the Glean SDK. The SDK takes care of storing & sending that data reliably.
-
-**Following [lean data practices](https://leandatapractices.com/) through SDK design choices.**
-It's easy to limit data collection to what's necessary and documentation can be generated easily, aiding both transparency & understanding for analysis.
-
-**Better data tooling integration due to standardized data types & registering them in machine-readable files.**
-By having collected data described in machine-readable files, our various data tools can read them and support metrics automatically, without manual work.
-
-**Due to common high-level concepts for metrics, APIs & data tools can better match the use-cases.**
-To make the choice easier for which metric type to use, we are introducing higher-level data types that offer clear and understandable semantics - for example, when you want to count something, you use the _"count"_ type. This also gives us opportunities to offer better tooling for the data, both on the client and for data tooling.
-
-**Basic semantics on how the data is collected are clearly defined by the library.**
-To make it easier to understand the general semantics of our data, the Glean SDK will define and document when which kind of data will get sent. This gives data analysis common basic semantics.
+The **Glean SDK** performs measurements and sends data from our products.
+It provides a set of **[metric types](https://mozilla.github.io/glean/user/metrics)** for individual measurements that are carefully designed to avoid common pitfalls with measurement.
+Metrics are then rolled up into **[pings](https://mozilla.github.io/glean/user/pings)** to send over the network.
+There are a number of built-in pings that are sent on predefined schedules, but it also possible to send custom pings at any desired cadence.
+
+The **Data Platform** validates and stores these pings in database tables.
+A fault tolerant design allows data to be retain in the event of problems such as traffic spikes or invalid data.
+Derived and cleaned data can also be automatically created at this stage.
+
+The **Analysis Tools** are used to query and visualize the data.
+Because Glean knows more about the individual data, such as its type and the ranges of acceptable values, it can in many cases provide the most appropriate visualization automatically.
+
+<!-- TODO: Link to GLAM -->
+
+# The Glean design principles
+
+**Provide a consistent base of telemetry**
+
+  A baseline of analysis is important for all our products, from counting active users to retention and session times. This is supported out-of-the-box by the SDK, and funnels directly into visualization tools like GUD.
```

Comment:
This is the first time GUD is mentioned: should we provide a link to the dashboard or to some explanation?